### PR TITLE
[217] Develop a Customer details list view page and Save Session in a Cookie

### PR DESCRIPTION
**Pull request by Israel-Force**
## Task: `Develop a Customer details list view page and Save Session in a Cookie`
- `added a folder for php and db task`
Screenshot (11)

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
Kindly add a screenshot of this output

File: submissions/israelOladosu/php-db/classes/dbh.class.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php 
+// class responsible for connecting to the Db
+class Dbh {
+
+    private $host = "localhost";
+    private $user = "root";
+    private $pwd = "mysql";
+    private $dbName = "cookiesDb";
```
Comment (adaugochi) [inline submissions/israelOladosu/php-db/classes/dbh.class.php:8]: Is this the name of the Database you created for your e-commerce task?

File: submissions/israelOladosu/php-db/index.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+include('./classes/queryDb.class.php');
+
+    // an instance of queryDb class to call the getAllUsers method;
+    $queryDb = new QueryDb();
+
+    $results = isset($_COOKIE['users'])? unserialize($_COOKIE['users']) : $queryDb->getAllUsers();
+
+    if(count($results) > 0) {
```
Comment (adaugochi) [inline submissions/israelOladosu/php-db/index.php:10]: Well done @Israel-Force But I wish you are not echoing all the time. There is a better way to do this

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/israelOladosu/php-db/classes/dbh.class.php (hunk 1 above)
Comment (Israel-Force) [reply submissions/israelOladosu/php-db/classes/dbh.class.php:8]: yes...

File: submissions/israelOladosu/php-db/index.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            echo "<tr>";
+                echo "<td>". $row['id'] . "</td>";
+                echo "<td>". $row['firstname'] . $row['lastname'] . "</td>";
+                echo "<td>". $row['email'] . "</td>";
+                echo "<td>". $row['created_at'] . "</td>";
+                echo '<td>'. 
+                        "<a href='#' >view </a>" . 
+                    '</td>';
+            echo "</tr>";
+};
```
Comment (adaugochi) [inline submissions/israelOladosu/php-db/index.php:29]: Kindly format from line 29 below properly

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**